File: web/entity/Customer.js
function Customer(customerId,customerName,customerAddress,customerSalary) {
    var __customerId =customerId;
    var __customerName =customerName;
    var __customerAddress =customerAddress;
    var __customerSalary =customerSalary;

    this.setCustomerId = function (customerId) {
        __customerId=customerId;
    }
    this.getCustomerId = function () {
        return __customerId;
    }
    this.setCustomerName = function (customerName) {
        __customerName=customerName;
    }
    this.getCustomerName = function () {
        return __customerName;
    }
    this.setCustomerAddress = function (customerAddress) {
        __customerAddress=customerAddress;
    }
    this.getCustomerAddress = function () {
        return __customerAddress;
    }
    this.setCustomerSalary = function (customerSalary) {
        __customerSalary=customerSalary;
    }
    this.getCustomerSalary = function () {
        return __customerSalary;
    }
}

File: BackEnd/src/entity/Customer.java
package entity;

public class Customer {
    private String customerID;
    private String customerName;
    private String customerAddress;
    private double customerSalary;

    public Customer() {
    }

    public Customer(String customerID, String customerName, String customerAddress, double customerSalary) {
        this.customerID = customerID;
        this.customerName = customerName;
        this.customerAddress = customerAddress;
        this.customerSalary = customerSalary;
    }

    public String getCustomerID() {
        return customerID;
    }

    public void setCustomerID(String customerID) {
        this.customerID = customerID;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getCustomerAddress() {
        return customerAddress;
    }

    public void setCustomerAddress(String customerAddress) {
        this.customerAddress = customerAddress;
    }

    public double getCustomerSalary() {
        return customerSalary;
    }

    public void setCustomerSalary(double customerSalary) {
        this.customerSalary = customerSalary;
    }
}
